academy/4-my-own-chain/game-winner.md

```diff
@@ -287,9 +287,7 @@ This should show:
 
 Testing with the CLI up to the point where the game is resolved with a rightful winner is better covered by unit tests or with a nice GUI. You will be able to partially test this in the [next section](./game-forfeit.md), via a forfeit.
 
-## Unit tests
-
-
+<!-- This headline should be deleted if no text follows ## Unit tests -->
```

Comment:
```suggestion
## Unit tests


```

Wait, this is already deleted as part of my merged PR. With this suggestion, I am trying to avoid a merge conflict.